packages/hydra-cli/src/codegen/WarthogWrapper.ts

```diff
@@ -29,11 +29,12 @@ export default class WarthogWrapper {
     this.prod = prod
     this.schemaResolvedPath = path.resolve(
       process.cwd(),
+      '../../',
```

Comment:
why do we need to go to the parent folder here? This seems a bit fragile. Ideally, `this.flags.schema` should work for both relative and absolute path

Reply:
Passing folder as the relative path was causing the confusing error and it was counterintuitive. However, this does break the absolute path so I'll revert it until we find fix for the relative path which doesn't break absolute path.